src/pages/Home/components/movie-modal-form/MovieModalForm.tsx

```diff
@@ -1,104 +1,170 @@
-import React, { FC, FormEvent } from 'react'
+import React, { FC, useState } from 'react'
+import { Modal } from '../../../../components/modal'
+import { IMovieModalForm } from './movieModalFormTypes'
+import styles from './MovieModalForm.module.scss'
 import classnames from 'classnames'
 import { Input } from '../../../../components/input'
 import { Select } from '../../../../components/select'
+import { FORM_VALIDATE, MOVIE_GENRES } from './movieModalFormConstans'
 import { TextArea } from '../../../../components/textarea'
 import { Button, ButtonTypes } from '../../../../components/button'
-import { Modal } from '../../../../components/modal'
-import { IMovieModalForm } from './movieModalFormTypes'
-import styles from './MovieModalForm.module.scss'
+import { useAction, useTypedSelector } from '../../../../hooks'
+import { useFormik } from 'formik'
+import { IMovieItem } from '../../../../store/actions-types'
+import { setEmptyMovieFields } from '../../../../utils'
+import { CompletionMoviePopup } from './CompletionMoviePopup'
 
-export const MovieModalForm: FC<IMovieModalForm> = ({modalName, formValues}) => {
 
-    const addMovieHandler = (event: FormEvent) => {
-        event.preventDefault()
-    }
+export const MovieModalForm: FC<IMovieModalForm> = ({modalName, formValues}) => {
+    const [addMovieSuccess, setAddMovieSuccess] = useState(false)
+    const {sortBy, filterBy} = useTypedSelector(state => state.movies)
+    const {addMovie, updateMovie, fetchMovies} = useAction()
+    const movieForm = useFormik({
+        initialValues: {
+            title: formValues?.title || '',
+            tagline: formValues?.tagline || '',
+            vote_average: formValues?.vote_average || '',
+            vote_count: formValues?.vote_count || '',
+            release_date: formValues?.release_date || '',
+            poster_path:  formValues?.poster_path || '',
+            overview:  formValues?.overview || '',
+            budget:  formValues?.budget || '',
+            revenue:  formValues?.revenue || '',
+            runtime:  formValues?.runtime || '',
+            genres:  formValues?.genres[0] || ''
+        } as IMovieItem,
+        validate: FORM_VALIDATE,
```

Comment:
Do not use uppercase for functions. Uppercase good for constants, for functions use just camelCase.

Reply:
renamed to movieFormValidate